app/Models/MiscModel.php

```diff
@@ -333,6 +333,13 @@ public function menuItems(array $items = []): array
                 'entity' => true,
                 'icon' => 'ra ra-fire-symbol',
             ];
+            if (auth()->check() && auth()->user()->can('update', $this)) {
```

Comment:
You can remove the link from the sidebar, there are already enough things there.